R/validate_submission.R

```diff
@@ -30,7 +30,9 @@
 #'
 #' ```{r, echo = FALSE}
 #' arrow::read_csv_arrow(system.file("check_table.csv", package = "hubValidations")) %>%
-#' dplyr::filter(.data$`parent fun` != "validate_model_metadata", !.data$optional) %>%
+#' dplyr::filter(
+#' !.data$`parent fun` %in% c("validate_model_metadata", "validate_target_file"),
+#' !.data$optional) %>%
 #'   dplyr::select(-"parent fun", -"check fun", -"optional") %>%
```

Comment:
Only the above block needs reviewing in this file. The rest of the changes are formatting.